diff --git bob/extension/test_extensions.py bob/extension/test_extensions.py
index e135217..58be65d 100644
--- bob/extension/test_extensions.py
+++ bob/extension/test_extensions.py
@@ -32,7 +32,7 @@ def _run(package, run_call):
     package_dir = os.path.join(temp_dir, "bob.example.%s" % package)

     # bootstrap
-    subprocess.call([sys.executable, "bootstrap-buildout.py"], cwd=package_dir, stdout=devnull, stderr=devnull)
+    subprocess.call([sys.executable, "bootstrap-buildout.py"], cwd=package_dir)
     assert os.path.exists(os.path.join(package_dir, "bin", "buildout"))

     # buildout
